docs/InstallEbpf.md

```diff
@@ -9,13 +9,24 @@ install or update the eBPF installation in the VM.
 Do the following from within the VM:
 
 1. Download the `eBPF-for-Windows.x.x.x.msi` file from the [latest release on GitHub](https://github.com/microsoft/ebpf-for-windows/releases).
-1. Execute the MSI file you downloaded.
-1. After accepting the License and selecting the desired installation folder (default will be "`C:\Program Files\ebpf-for-windows`"), the following components will be selectable from the *Installation Wizard*:
+2. For Debug EBPF variant with JIT, below debug DLL are required to be present on test system on a location included in %PATH% so that they can be loaded. These debug DLL are provided along with Visual Studio under C:\Windows\System32 -
```

Comment:
I don't think this is a good direction to go.  The install script should take care of it for the user, who should not need to do anything themselves.

Reply:
the script helps the user copy the debug DLL to the JIT subfolder under the ebpf install directory as part of this fix - https://github.com/microsoft/ebpf-for-windows/pull/3957/commits/7968287534eb0de5ac378f32845087b3ebeeec7e This documentation is concerned only with the case where the customer is using just the debug MSI to install ebpf

Reply:
Yes, ideally we would delete the script and just use the MSI for developers.